client/src/components/Login.js

```diff
@@ -173,10 +181,18 @@ const Login = (props) => {
             type="password"
             id="password"
             autoComplete="current-password"
+            onFocus={() => {
+              setPasswordFocus(true)
+            }}
+            onBlur={() => {
+              setPasswordFocus(false)
```

Comment:
In my opinion, we should not set it back to false, because after user have touched the input, we should show the error message, even though the user blurs the input.